features/support/email_alert_signup_helper.rb

```diff
@@ -0,0 +1,26 @@
+require 'ostruct'
+
+module EmailAlertSignupHelper
+  include GovukContentSchemaExamples
+
+  def content_store_has_employment_email_alert_signup(base_path:, tags:)
+    content_store_has_item(base_path, govuk_content_schema_example("email_alert_signup").merge(tags).to_json)
+  end
+
+  def expect_registration_to(title:, tags:, base_path:)
```

Comment:
This expectation appears to be setting a stub rather than an actual expectation. Does that mean the scenario will always pass, even if the subscription never happens?